manager/state/raft/raft.go

```diff
@@ -270,6 +275,9 @@ func (n *Node) JoinAndStart() error {
 			n.Node = raft.StartNode(n.Config, []raft.Peer{})
 
 			if err := n.registerNodes(resp.Members); err != nil {
```

Comment:
Err - sorry, I missed the `return err`.

I always get confused by named returns - they're a bit error prone (at least to me)